cms/templates/import.html

```diff
@@ -113,11 +165,17 @@ <h3 class="title">${_("Updating Course")}</h3>
 
               <div class="status-detail">
                 <h3 class="title">${_("Success")}</h3>
-                <p class="copy">${_("Your imported content has now been integrated into this course")}</p>
+                <p class="copy">
+                    %if library:
+                        ${_("Your imported content has now been integrated into this library")}
+                    %else:
+                        ${_("Your imported content has now been integrated into this course")}
+                    %endif
+                </p>
 
                 <ul class="list-actions">
                   <li class="item-action">
-                    <a href="${successful_import_redirect_url}" class="action action-primary">${_("View Updated Outline")}</a>
+                    <a href="${successful_import_redirect_url}" id="view-updated-button" class="action action-primary">${_("View Updated Outline")}</a>
```

Comment:
@Kelketek Per discussion with @marcotuts in the main conversations, can we change this to "View Updated Library" please